musicname/ze_ouroboros_v3.cfg

```
@@ -0,0 +1,8 @@
+"music"
+{
+    "the caretaker - everywhere at the end of time - 35 f5 - aching cavern without lucidity.mp3"            "The Caretaker - Everywhere at The End of Time - 35 f5 - Aching CAvern without Lucidity"
```

Typo, capital A in `CAvern`
```suggestion
    "the caretaker - everywhere at the end of time - 35 f5 - aching cavern without lucidity.mp3"            "The Caretaker - Everywhere at The End of Time - 35 f5 - Aching Cavern without Lucidity"
```